executor/slow_query.go

```diff
@@ -373,13 +384,16 @@ func (st *slowQueryTuple) setFieldValue(tz *time.Location, field, value string,
 		}
 	case variable.SlowLogTxnStartTSStr:
 		st.txnStartTs, err = strconv.ParseUint(value, 10, 64)
-	case variable.SlowLogUserStr:
```

Comment:
Does this case need to be kept for the capability of old format?

Reply:
updated. PTAL.